From d978e784f433346d3676b5de805b3cea36b835c4 Mon Sep 17 00:00:00 2001
From: Tomi Valkeinen <tomi.valkeinen@ideasonboard.com>
Date: Fri, 29 Sep 2023 16:23:58 +0300
Subject: [PATCH] media: rp1: cfe: Expose find_format_by_pix()

Make find_format_by_pix() accessible to other files in the driver.

Signed-off-by: Tomi Valkeinen <tomi.valkeinen@ideasonboard.com>
---
 drivers/media/platform/raspberrypi/rp1_cfe/cfe.c | 2 +-
 drivers/media/platform/raspberrypi/rp1_cfe/cfe.h | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/media/platform/raspberrypi/rp1_cfe/cfe.c
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/cfe.c
@@ -461,7 +461,7 @@ const struct cfe_fmt *find_format_by_cod
 	return NULL;
 }
 
-static const struct cfe_fmt *find_format_by_pix(u32 pixelformat)
+const struct cfe_fmt *find_format_by_pix(u32 pixelformat)
 {
 	unsigned int i;
 
--- a/drivers/media/platform/raspberrypi/rp1_cfe/cfe.h
+++ b/drivers/media/platform/raspberrypi/rp1_cfe/cfe.h
@@ -36,5 +36,6 @@ extern const struct v4l2_mbus_framefmt c
 extern const struct v4l2_mbus_framefmt cfe_default_meta_format;
 
 const struct cfe_fmt *find_format_by_code(u32 code);
+const struct cfe_fmt *find_format_by_pix(u32 pixelformat);
 
 #endif
